src/js/user-profile/containers/UserProfileApp.jsx

```diff
@@ -1,18 +1,39 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import moment from 'moment';
 
+import { removeSavedForm } from '../actions/index';
 import UserDataSection from '../components/UserDataSection';
 import AuthApplicationSection from '../components/AuthApplicationSection';
-
+import FormList from '../components/FormList';
 import RequiredLoginView from '../../common/components/RequiredLoginView';
 
+import { isSIPEnabledForm } from '../helpers';
+
+moment.updateLocale('en', {
+  meridiem: (hour) => {
+    if (hour < 12) {
+      return 'a.m.';
+    }
+    return 'p.m.';
+  }
+});
+
 class UserProfileApp extends React.Component {
   render() {
+    const savedForms = this.props.profile.savedForms;
+    const verifiedSavedForms = savedForms.filter(isSIPEnabledForm);
+    const hasVerifiedSavedForms = !!verifiedSavedForms.length;
+
     const view = (
       <div className="row user-profile-row">
         <div className="usa-width-two-thirds medium-8 small-12 columns">
           <h1>Your Account</h1>
           <div>
+            {hasVerifiedSavedForms && <FormList
```

Comment:
Can you add a build type check here so it's not visible in production yet? Per https://dsva.slack.com/archives/C446YB1K8/p1504101843000103

Reply:
Good idea, added. @rtluu should we include removing this to the testing ticket created for this?